### PR TITLE
Console SQL Shell 

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-1935

Created Limited Access page for the in-Console SQL Shell
Added to Cloud Deployments section of Nav
Release note

### CODE REVIEW

**Comment by gemma-shay** [comment]
@Amruta-Ranade just tagging in for any input on the page layout. also @nickvigilante am I doing the links right (fully qualified for going either way between cloud and stable)?

File: src/current/_includes/releases/cloud/2023-09-05.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## September 05, 2023
+
+<h3 id="2023-09-05-general-changes"> General changes </h3>
+
+- The {{ site.data.products.cloud }} Console's [SQL Shell]({% link cockroachcloud/sql-shell.md %}) is now available in [limited access](https://cockroachlabs.com/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html). This feature enables you to run queries on your cluster directly from the {{ site.data.products.cloud }} Console. To enroll your organization, contact your Cockroach Labs account team.
```
Comment (nickvigilante) [inline src/current/_includes/releases/cloud/2023-09-05.md:5]:
```suggestion
- The {{ site.data.products.cloud }} Console's [SQL Shell]({% link cockroachcloud/sql-shell.md %}) is now available in [limited access](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroachdb-feature-availability). This feature enables you to run queries on your cluster directly from the {{ site.data.products.cloud }} Console. To enroll your organization, contact your Cockroach Labs account team.
```

File: src/current/cockroachcloud/sql-shell.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## Limitations
+
+- All statements in the SQL Shell are executed within a transaction, so queries like [`SET CLUSTER SETTING`](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/set-cluster-setting.html) are not supported.
+- The SQL Shell does not yet support [sessions](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/sessions.html).
+- The SQL Shell is available only to users of enrolled organizations with the [Cluster Administrator or Cluster Operator role]({% link cockroachcloud/managing-access.md %}).
+
+## Overview
+
+Above the SQL Shell input field, you will see the active user and cluster details in the format `{user name} @ {cluster-name}:{database}`. Note that the user is the **Team member** you are logged into the {{ site.data.products.cloud }} Console as, not a SQL user, and you must have the Cluster Administrator or Cluster Operator [role]({% link cockroachcloud/managing-access.md %}) to use this SQL Shell. Other users can still access CockroachDB's [command line SQL shell](https://cockroachlabs.com/docs/{{site.versions["stable"]}}/cockroach-sql.html).
```
Comment (nickvigilante) [inline src/current/cockroachcloud/sql-shell.md:25]:
```suggestion
Above the SQL Shell input field, you will see the active user and cluster details in the format `{user name} @ {cluster-name}:{database}`. Note that the user is the **Team member** you are logged into the {{ site.data.products.cloud }} Console as, not a SQL user, and you must have the Cluster Administrator or Cluster Operator [role]({% link cockroachcloud/managing-access.md %}) to use this SQL Shell. Other users can still access CockroachDB's [command line SQL shell](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroach-sql).
```

File: src/current/cockroachcloud/sql-shell.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+To execute a SQL statement, enter it in the input field and either click **Run** or use the **Enter** key. The statement status will be **Loading** until it either **Succeeds** or returns an **Error**. Any results returned can be exported by clicking the **Export results** button below the executed statement.
+
+You can select any statement that you've previously run and copy it, edit it, or re-run it.
+
+## Example workflow
+
+The following examples assume you have already [created a CockroachDB {{ site.data.products.cloud }} cluster]({% link cockroachcloud/create-a-serverless-cluster.md %}) and have [access](#limitations) to the SQL Shell.
+
+1. In the SQL Shell, run [`CREATE TABLE`](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/create-table.html) followed by a table name, the column names, and the [data type](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/data-types.html) and [constraint](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/constraints.html), if any, for each column:
```
Comment (nickvigilante) [inline src/current/cockroachcloud/sql-shell.md:37]:
```suggestion
1. In the SQL Shell, run [`CREATE TABLE`](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/create-table) followed by a table name, the column names, and the [data type](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/data-types) and [constraint](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/constraints), if any, for each column:
```

File: src/current/cockroachcloud/sql-shell.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      2  | 2 | 25000.00
+    ~~~
+    
+    Note that each line of a query's results will be numbered independently of the output. This is for readability and will not be shown in any exported data.
+
+1. Click **Export results** to download a CSV file of the output.
+
+## See also
+
+- [`cockroach sql`](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroach-sql.html)
```
Comment (nickvigilante) [inline src/current/cockroachcloud/sql-shell.md:97]:
```suggestion
- [`cockroach sql`](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroach-sql)
```

File: src/current/cockroachcloud/sql-shell.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+- [Learn CockroachDB SQL](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/learn-cockroachdb-sql.html)
```
Comment (nickvigilante) [inline src/current/cockroachcloud/sql-shell.md:98]:
```suggestion
- [Learn CockroachDB SQL](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/learn-cockroachdb-sql)
```

File: src/current/cockroachcloud/sql-shell.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+The **SQL Shell** page on the {{ site.data.products.db }} Console enables you to run queries on your cluster directly from your browser.
+
+To use this feature, select a cluster from the [**Clusters** page](cluster-management.html#view-clusters-page), and navigate to the cluster's **SQL Shell** page.
+
+## Limitations
+
+- All statements in the SQL Shell are executed within a transaction, so queries like [`SET CLUSTER SETTING`](https://cockroachlabs.com/docs/{{ site.current_cloud_version }}/set-cluster-setting.html) are not supported.
```
Comment (nickvigilante) [inline src/current/cockroachcloud/sql-shell.md:19]:
```suggestion
- All statements in the SQL Shell are executed within a transaction, so queries like [`SET CLUSTER SETTING`](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/set-cluster-setting) are not supported.
```

**Review by nickvigilante — APPROVED** [review]
LGTM with a few changes! We generally want this format in order to avoid 3XX status codes completely:

```
https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/path-to-page-with-no-html-extension
```

**Review by Amruta-Ranade — COMMENTED** [review]
The structure looks good to me

File: src/current/cockroachcloud/sql-shell.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## Limitations
+
+- All statements in the SQL Shell are executed within a transaction, so statements like [`SET CLUSTER SETTING`](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/set-cluster-setting) are not supported.
+- The SQL Shell does not yet support sessions.
+- The SQL Shell is available only to users of enrolled organizations with the [Cluster Administrator or Cluster Operator role]({% link cockroachcloud/managing-access.md %}).
+
+## Overview
+
+Above the SQL Shell input field, you will see the active user and cluster details in the format `{user name} @ {cluster-name}:{database}`. Note that the user is the **Team member** currently logged into the {{ site.data.products.cloud }} Console, not a SQL user, and you must have the Cluster Administrator or Cluster Operator [role]({% link cockroachcloud/authorization.md %}#organization-user-roles) to use this SQL Shell. Other users can still access CockroachDB's [command line SQL shell](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroach-sql).
```
Comment (Amruta-Ranade) [inline src/current/cockroachcloud/sql-shell.md:25]: We can specify which SQL user is being used (is it `root`?)

Comment (Amruta-Ranade) [inline src/current/cockroachcloud/sql-shell.md:25]:
```suggestion
Above the SQL Shell input field, you will see the active user and cluster details in the format `{user name} @ {cluster-name}:{active database}`. Note that the user is the **Team member** currently logged into the {{ site.data.products.cloud }} Console, not a SQL user, and the Team member must have the Cluster Administrator or Cluster Operator [role]({% link cockroachcloud/authorization.md %}#organization-user-roles) to use this SQL Shell. Other Team members can still access CockroachDB's [command line SQL shell](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroach-sql).
```

**Review by Amruta-Ranade — APPROVED** [review]

**Review by lassenordahl — APPROVED** [review]

File: src/current/cockroachcloud/sql-shell.md (hunk 7 above)
Comment (lassenordahl) [reply src/current/cockroachcloud/sql-shell.md:25]: Yes

**Pull request merged**